packages/server/src/api.ts

```diff
@@ -45,8 +45,14 @@ export class Api {
     this.root.get('/version', this.version.bind(this))
     this.root.get('/status', this.status.bind(this))
     this.root.use('/api', this.router)
-    this.router.use(cors())
-    this.router.use(express.json())
+    this.router.use(
+      cors({
+        origin: '*',
+        methods: ['GET', 'POST', 'DELETE'],
+        allowedHeaders: ['Content-Type', 'Authorization', 'Content-Length']
+      })
+    )
```

Comment:
What does this do?

Reply:
This simply strengthens the CORS for our API. It basically tells that allowed requests from any origin, that we only allow GET, POST, and DELETE, and some headers. This is mostly for requests coming from a browser but it still good practice to be as restrictive as possible.